lib/chrome.js

```diff
@@ -88,7 +88,7 @@ export class StorageArea {
 
   get(keys, callback) {
     const this_items = this.items;
-    const records = [];
+    const records = {};
```

Comment:
No (or we would have to cast it to an `Object` later), since we need to give an ['Object with items in their key-value mappings'](https://developer.chrome.com/extensions/storage) to the callback.